scalafmt-dynamic/src/main/scala/org/scalafmt/dynamic/ScalafmtDynamicDownloader.scala

```diff
@@ -91,7 +91,7 @@ class ScalafmtDynamicDownloader(
         "https://oss.sonatype.org/content/repositories/snapshots"
       ),
       MavenRepository.of("https://oss.sonatype.org/content/repositories/public")
-    )
+    ) ++ customRepositories
```

Comment:
i think custom repos should either override the defaults, or have higher priority - one usecase of this (ours) is that oss repositories are disabled and must go through our own artifactory and it causes timeouts when trying to download stuff from OSS ... therefore it makes sense to try custom repositories first and fallback on defaults

Reply:
timeouts are good point, will move it in front then